src/main/java/bio/terra/cli/service/WorkspaceManagerService.java

```diff
@@ -134,28 +136,15 @@ public WorkspaceManagerService(@Nullable AccessToken accessToken, Server server)
    * @return the Workspace Manager version object
    */
   public SystemVersion getVersion() {
-    UnauthenticatedApi unauthenticatedApi = new UnauthenticatedApi(apiClient);
-    try {
-      return HttpUtils.callWithRetries(
-          () -> unauthenticatedApi.serviceVersion(), WorkspaceManagerService::isRetryable);
-    } catch (ApiException | InterruptedException ex) {
-      throw new SystemException("Error getting Workspace Manager version", ex);
-    }
+    return callWithRetries(
+        new UnauthenticatedApi(apiClient)::serviceVersion,
+        "Error getting Workspace Manager version");
   }
 
   /** Call the Workspace Manager "/status" endpoint to get the status of the server. */
   public void getStatus() {
-    UnauthenticatedApi unauthenticatedApi = new UnauthenticatedApi(apiClient);
-    try {
-      HttpUtils.callWithRetries(
-          () -> {
-            unauthenticatedApi.serviceStatus();
-            return null;
-          },
-          WorkspaceManagerService::isRetryable);
-    } catch (ApiException | InterruptedException ex) {
-      throw new SystemException("Error getting Workspace Manager status", ex);
-    }
+    callWithRetries(
+        new UnauthenticatedApi(apiClient)::serviceStatus, "Error getting Workspace Manager status");
```

Comment:
curious: will this create a new client for every retry, or reuse the same one?

Reply:
This definitely reuses the same `ApiClient` for every retry, and I think also the same `UnauthenticatedApi`.